cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,24 @@
+import {SauceContentHomePage, SauceStandardLogin, SauceInventory, AddProductToCart, SauceCart, SauceCheckoutInformation,
+  CheckoutOverview, FinishCheckout} from "../page/index";
+
+const menuContentPage = new SauceContentHomePage();
+const LoginPage = new SauceStandardLogin();
+const InventoryPage = new SauceInventory();
+const InventoryDetailsPage = new AddProductToCart();
+const ShoppingCartPage = new SauceCart();
+const CheckoutInformationPage = new SauceCheckoutInformation();
+const CheckoutOverviewPage = new CheckoutOverview();
+const CheckoutCompletePage = new FinishCheckout();
+
 describe("Buy a black t-shirt", () => {
   it("then the t-shirt should be bought", () => {
-    cy.visit("https://www.saucedemo.com/"); // (1)
-    cy.get(".login-box > form > div > input").first().type("standard_user"); // (2)
-    cy.get(".login-box > form > div > input").last().type("secret_sauce"); // (2)
-    cy.get("input[type='submit']").click(); // (2)
-    cy.get("#item_1_title_link").click();// (3)
-    cy.get("#add-to-cart-sauce-labs-bolt-t-shirt").click();// (4)
-    cy.get("a[class='shopping_cart_link']").click();// (5)
-    cy.get("button[id='checkout']").click();// (6)
-    cy.get("#first-name").type("Cypress");// (7)
-    cy.get("#last-name").type("Workshop");// (8)
-    cy.get("#postal-code").type("00000");// (9)
-    cy.get("input[id='continue']").click();// (10)
-    cy.get("button[id='finish']").click();// (11)
-    cy.get("#contents_wrapper > .checkout_complete_container > h2").should(
-        "have.text",
-        "Thank you for your order!",
-    ); // (12)
+    menuContentPage.visitMenuContentPage(); // (1)
+    LoginPage.enterStandardUserCredentials(); // (2)
+    InventoryPage.openProductDetails();// (3)
+    InventoryDetailsPage.addProductToCart(); // (4)(5)
+    ShoppingCartPage.proceedToCheckout(); // (6)
+    CheckoutInformationPage.enterUserInformation(); // (7)(8)(9)(10)
+    CheckoutOverviewPage.acceptCheckoutDetails(); // (11)
+    CheckoutCompletePage.confirmSuccessfulCheckout(); // (12)
```

Comment:
please pass as a input parameter the message to be verified by confirmSuccessfulCheckout method